pyiron_base/job/generic.py

```diff
@@ -512,6 +512,10 @@ def copy_to(self, project=None, new_job_name=None, input_only=False, new_databas
         # Get new hdf location
         project = project or self.project
         new_job_name = new_job_name or self.job_name
+        job_table = project.job_table(recursive=False)
+        if len(job_table) > 0:
+            if new_job_name in job_table.job.tolist():
+                return project[new_job_name]
```

Comment:
```suggestion
                return project.load(new_job_name)
```
To be a bit more explicit.